Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/CollectionViews/ListCollectionView.cs

```diff
@@ -90,7 +90,9 @@ protected override void RefreshOverride()
             if (!UsesLocalArray)
             {
 #endif
-                _internalList = list;
+#pragma warning disable SA1137 // Elements should have the same indentation
+            _internalList = list;
+#pragma warning restore SA1137 // Elements should have the same indentation
```

Comment:
Is there a reason we're not indenting? I'm confused.

Reply:
Look around it. It's inside a block. Or not, since it's on a conditional compilation block. I decided to disable it only there. If we indent it, then it will look wrong when the flag is on/off.